### PR TITLE
Always print duration in the same format and units

### PR DESCRIPTION
**Pull request by cescgina**
There were some small divergences in how build and test durations were
printed. This changes makes all time durations print in minutes and
using 2 decimal places.

### CODE REVIEW

File: cibyl/utils/time.py [hunk 1]
```diff
@@ -15,10 +15,15 @@
...
+    :param duration: The duration to convert.
+    :param unit: The unit the duration is in.
     :return: The time in minutes.
     """
-    return ms / 60000
+    if unit == "ms":
+        return duration / 60000
+    else:
+        # assume seconds as default unit
+        return duration / 60
```
Comment (jsanemet) [inline cibyl/utils/time.py:29]: If the unit is unknown, it is better practice to raise a NotImplementedError. The documentation says nothing about the assumption, so it is an unexpected behavior waiting to happen.

**Review by jsanemet — CHANGES_REQUESTED** [review]
Everything looks good, I just have a comment on the 'as_minutes' function.

File: cibyl/utils/time.py (hunk 1 above)
Comment (cescgina) [reply cibyl/utils/time.py]: Good point, I've changed it

**Review by jsanemet — APPROVED** [review]
All good, awesome.

**Pull request merged**